### ISSUE TITLE
Performance better than that in the paper. 

### ISSUE THREAD

**Issue by chenyangh** [rstrudel/segmenter#14]
Hi Robin, 

Thanks for releasing the code and model. I find that your model performs better than what is reported in the paper. For example, on ADE20K validation set, Seg-B-Mask/16 has 45.69 mIoU (SS), but according to the information from this repo, it can actually achieve 48.5. Am I missing something?

**Comment by woctezuma** [comment]
Maybe #7:

> We plan on updating the paper to be consistent with the new reported results.

**Comment by chenyangh** [comment]
> Maybe #7:
> 
> > We plan on updating the paper to be consistent with the new reported results.

Thanks. It is surprising that the new version has such huge improvements.

**Comment by rstrudel** [comment]
Hi @chenyangh ,

Indeed, as explained in #7 , the use of improved ViT pretrained weights overall improves our final performance on semantic segmentation. We updated all the tables in the new version of the paper which should be on arXiv in 2 days maximum. You can then check Section 4.2 ImageNet pre-training where we shortly discuss the difference of performance we get from the original ViT checkpoints versus the Steiner et al ViT checkpoints.

**Comment by rstrudel** [comment]
The updated version is now on arxiv.
